config/index.js

```diff
@@ -244,32 +244,6 @@ var conf = convict({
         env: 'PREPEND_VERIFICATION_SUBDOMAIN_SUBDOMAIN'
       },
     },
-    verificationUrl: {
```

Comment:
Can you add this list to the deploy doc to notify @jrgm in case they are still in any config files?

Reply:
yea!